src/sas/qtgui/Utilities/Preferences/PlottingPreferencesWidget.py

```diff
@@ -53,3 +60,4 @@ def _restoreFromConfig(self):
         self.legendLineLength.setStyleSheet("background-color: white")
         self.disableResidualPlot.setChecked(config.DISABLE_RESIDUAL_PLOT)
         self.disablePolydispersityPlot.setChecked(config.DISABLE_POLYDISPERSITY_PLOT)
+        self.useMatplotlibToolbar.setChecked(config.USE_MATPLOTLIB_PLOT_TOOLBAR)
```

Comment:
```suggestion
        self.useMatplotlibToolbar.setChecked(config.USE_MATPLOTLIB_TOOLBAR)
```

Opening a plot after the preference is selected throws an Exception because of this line.

Reply:
I overlooked this, had the variable named differently before. thanks for correcting